docs/jobs.md

```diff
@@ -49,25 +50,27 @@ workflows:
 ```
 
 ```yaml
-# .circleci/orbs/hello-orb/jobs/sayhello.yml
-parameters:
-  saywhat:
-    description: "To whom shall we say hello?"
-    default: "World"
-    type: string
-machine: true
-steps:
-  - say:
-      saywhat: "<< parameters.saywhat >>"
-```
+# .circleci/orbs/hello-orb/orb.yml
+name: hello-orb
 
-```yaml
-# .circleci/orbs/hello-orb/commands/say.yml
-parameters:
+jobs:
+  sayhello:
+    parameters:
+      saywhat:
+        description: "To whom shall we say hello?"
+        default: "World"
+        type: string
+    machine: true
+    steps:
+      - say:
+          saywhat: "<< parameters.saywhat >>"
+commands:
   saywhat:
-    type: string
-steps:
```

Comment:
This needs to stay, no?

Reply:
It's still there, just at a different level of indentation